requirements/edx/base.txt

```diff
@@ -471,9 +471,9 @@ edx-django-utils==5.4.0
     #   ora2
     #   outcome-surveys
     #   super-csv
-edx-drf-extensions==8.8.0
+edx-drf-extensions @ git+https://github.com/openedx/edx-drf-extensions.git@umar/make-verify-signature-method-generic
```

Comment:
📢📢📢
This change is just for testing.
And only related to this PR.
Will be revered in this PR before merging.